pages/services/data-science-engine/1.0.0/examples/index.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: layout.pug
+navigationTitle: Examples
+excerpt: Usage examples
+title: Install
+menuWeight: 7
+model: /services/data-science-engine/data.yml
+render: mustache
+---
+This section contains usage examples for using {{ model.techName }}.
+
+# Basic
+
+Perform a default installation by following the instructions in the [Install and Customize](/services/data-science-engine/1.0.0/install/) section.
+
+## Run a {{ model.packageName }} job
```

Comment:
None of these will apply. Anything that says
```
dcos {{packagename}} run --submit-args
```
That's all Spark. Inside of DSEngine they'll run spark-submit commands like normal `spark-submit`

Reply:
You'll want example notebooks users can run. 
Such as these: 
- https://github.com/mesosphere/jupyter-service/blob/master/tests/notebooks/launch_spark_example.ipynb
- https://github.com/mesosphere/jupyter-service/blob/master/tests/notebooks/tensorflow_mnist.ipynb

Reply:
Thanks. Those URLs include references to `jupyter-service`. We okay with that?

Reply:
These are good to have, but I'm going to need more detail than this. The raw code doesn't give me enough context for the user. Do you have something more descriptive to share?

Reply:
You can look this PR (https://github.com/mesosphere/dcos-docs-site/pull/2452).